docker_scripts/windows_start.bat

```diff
@@ -0,0 +1,2 @@
+start http://localhost:3000
+docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -p 3000:3000 -p 4000:4000 wpilib/axon:latest
```

Comment:
I think we should commit this file with CRLF line endings that way Windows users who don't have autocrlf enabled do not need to convert the file first.

Reply:
My local version has them now. Saving for a future PR.